src/helpers/handlers.js

```diff
@@ -39,3 +41,12 @@ export const isUkrainianPage = () => {
   const pathname = useLocation().pathname;
   return pathname.includes(`/ua/`);
 };
+
+export const getTranslatedText = (path) => {
```

Comment:
move this helper function to separate file (utils dir)

Reply:
https://github.com/adaptdk/su-ukraina/pull/198/commits/b71faf36fcf9b22e3367ca5b993363b12cebf167